### DISCUSSION TITLE
include irvine assembly

### DISCUSSION THREAD

**Discussion by F998r** [Homebrew/discussions#3685] [Getting started]
hello guys 
I am studying assembly course this semester
unfortunately my os is mac so I installed win10 through bootcamp 
and install vs2022 
Just the tutorial project works 
all my projects raise errors
I can't include Irvine32 neither Irvine64 lib 
literally I saw all web tutorials and issues on Stack overflow 
so plz HELP!!

**Comment by SMillerDev** [comment]
I'm not sure what homebrew has to do with this
